### PR TITLE
Proj1702: Omit XML declarations

### PR DESCRIPTION
**Pull request by Corniel**
Documentation for https://github.com/dotnet-project-file-analyzers/dotnet-project-file-analyzers/pull/162

### CODE REVIEW

File: rules/Proj1702.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Proj1701: Omit XML declarations
+The XML Declaration provides basic information about the format for the rest of
+the XML document. It takes the form of a Processing Instruction and can have
+the attributes version, encoding and standalone. All of this is redunent for
```
Comment (LauraXiulan) [inline rules/Proj1702.md:4]: What is the "standalone"?

Comment (LauraXiulan) [inline rules/Proj1702.md:4]: redundant

**Review by LauraXiulan — APPROVED** [review]
See comments

File: rules/Proj1702.md (hunk 1 above)
Comment (LauraXiulan) [inline rules/Proj1702.md]:
```suggestion
the attributes version, encoding and standalone. All of this is redundant for
```

Comment (Corniel) [reply rules/Proj1702.md]: It means that the data in the XML does not need external sources (other XML/schema) to be understood. Also totally irrelevant in this context.

**Pull request merged**